ephios/plugins/complexsignup/templates/complexsignup/fragment_participation.html

```diff
@@ -3,17 +3,16 @@
 {% load user_extras %}
 {% load i18n %}
 
-{% block participation_controls %}
-    <div class="mb-n3">
-        {% crispy_field form.unit_path label_class="d-none" wrapper_class="d-inline-block me-3" %}
-    </div>
-{% endblock %}
-
 {% block participation_detail %}
     {% if form.preferred_unit_name %}
         <small class="text-body-secondary">
             <br> {% translate "prefers" %}
             {{ form.preferred_unit_name }}
         </small>
     {% endif %}
+    </div> {# end previous column, start a new one #}
+    <div class="col-12 col-md-4 col-lg-12 col-xxl-4 d-flex justify-content-between align-items-top">
```

Comment:
<img width="421" height="201" alt="image" src="https://github.com/user-attachments/assets/57044ba4-7e0c-4cf2-91aa-2c9d12d9b9bf" />
i feel like there is a bit too much padding to the right?

Reply:
I can check again if I can make the outer right column smaller. Content can't flow under the cogwheel, that's its own column.

Reply:
yeah thats expected, my point was more that the cogwheel should be closer to the right border of the box in my perception

Reply:
I lowered the gutter to a point where it still looked okay. The button with the cogwheel is a little larger because it has room for an extra exclamation mark.
<img width="489" height="717" alt="image" src="https://github.com/user-attachments/assets/0dc1c127-07dd-4de9-bfe7-7fa9e354985d" />